theme.qml

```diff
@@ -269,6 +285,14 @@ FocusScope {
             menu_games:"Spiele"
         },
         "fr": {
+            global_dimm: "Assombrir",
+            global_bright: "Illuminer",
+            global_Dark: "Sombre",
+            global_Light: "Clair",
+            global_OzoneDark: "OzoneDark",
+            global_SteamOS: "SteamOS",
+            global_mono: "Monochrome",
+            global_color: "Couleur",

```

Comment:
```suggestion
            settings_collection_accentColor_dimm: "Assombrir",
            settings_collection_accentColor_bright: "Illuminer",
            settings_general_colorScheme_Dark: "Sombre",
            settings_general_colorScheme_Light: "Clair",
            settings_general_colorScheme_OzoneDark: "OzoneDark",
            settings_general_colorScheme_SteamOS: "SteamOS",
            settings_general_logoVariant_mono: "Monochrome",
            settings_general_logoVariant_color: "Couleur",
```
global is for text strings which can appear in many places in the theme. I assume these strings will never be needed outside the settings screen. Therefore I would have adapted the naming scheme to this one

Reply:
ok good information, i share this with my friends in our discord channel #Shinretro-theme :)